src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2362,79 +3800,46 @@
-        For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
+    short-summary: Delete a packet capture session.
     examples:
         - name: Delete a packet capture session. Note that this only deletes the session and not the capture file.
-          text: az network watcher packet-capture delete -n packetCaptureName --location westcentralus
+          text: az network watcher packet-capture delete -n packetCaptureName -l westcentralus
 """
 
 helps['network watcher packet-capture list'] = """
     type: command
-    long-summary: >
-        This command requires that both Azure Network Watcher is enabled for the VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
-        For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
-    examples:
-        - name: List all packet capture sessions within a resource group.
-          text: az network watcher packet-capture list --location westus
-"""
-
-helps['network watcher packet-capture show'] = """
-    type: command
-    long-summary: >
-        This command requires that both Azure Network Watcher is enabled for the VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
-        For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
-    examples:
-        - name: Show a packet capture session.
-          text: az network watcher packet-capture show --location westus -n MyPacketCapture
-"""
-
-helps['network watcher packet-capture show-status'] = """
-    type: command
-    long-summary: >
-        This command requires that both Azure Network Watcher is enabled for the VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
-        For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
-    examples:
-        - name: Show the status of a packet capture session.
-          text: az network watcher packet-capture show-status --location westus -n MyPacketCapture
-"""
-
-helps['network watcher packet-capture stop'] = """
-    type: command
-    long-summary: >
-        This command requires that both Azure Network Watcher is enabled for the VM's region and that AzureNetworkWatcherExtension is enabled on the VM.
-        For more information about configuring packet capture visit https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-packet-capture-manage-cli
+    short-summary: List all packet capture sessions within a resource group (note --location parameter since one Network Watcher per region).
```

Comment:
Remove parenthetical. If this information must be included, add as part of `long-summary`.